site/docs/sql/file-connector.md

```diff
@@ -29,23 +29,23 @@ above is considered local (i.e. files residing on Jet members), e.g.
 `/path/to/directory/`.
 
 `glob` is a pattern to filter the files in the specified directory.
-The default value is '*', matching all files.
+The default value is `*`, matching all files.
 
-## Ignore File Not Found
+### Ignore File Not Found
 
-When you create a mapping without a schema, or when you execute a
-`SELECT` query, the location specified by the `path` option is expected
-to contain at least one file matching the `glob`, otherwise an exception
-is thrown. This is to avoid hard to catch mistakes, such as typos. If
-you simply want to return zero results set the `ignoreFileNotFound`
-option to true. Note that in this case you must specify the schema.
+When you create a mapping without a schema, the location specified by the
```

Comment:
The term "schema" is quite confusing in sql. It's an actual object in some DBs, in calcite it denotes a hierarchical collection of objects etc. I'd rather avoid it and use this:
```suggestion
When you create a mapping without a column list, the location specified by the
```